react/training/src/app/screens/Login/components/layout.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { Field, reduxForm, Form } from 'redux-form';
+import PropTypes from 'prop-types';
+
+import { required, validEmail, passwordLength } from './utils';
+import { CustomInput } from './CustomInput/index';
+
+class LoginForm extends Component {
+  render() {
+    const { handleSubmit } = this.props;
+    return (
+      <Form onSubmit={handleSubmit}>
+        <Field
+          name="mail"
+          component={CustomInput}
+          type="text"
+          label="Email"
+          validate={[required, validEmail]}
+        />
+        <Field
+          name="password"
+          component={CustomInput}
+          type="password"
+          label="Password"
+          validate={[required, passwordLength]}
+        />
+        <button type="submit">Submit</button>
```

Comment:
There is no need to do this in this pr. We are going to handle the login redirection to the `Game` in the next pr.